libs/sdk-ui-pivot/src/impl/utils.ts

```diff
@@ -35,3 +44,21 @@ export async function sleep(delay: number): Promise<void> {
         setTimeout(resolve, delay);
     });
 }
+
+/**
+ * Remove invalid totals from an execution definition given a list of sort items
+ *
+ * @param def - an execution definition to sanitize
+ * @param sortItems - a specification of the sort
+ */
+export function sanitizeDefTotals(def: IExecutionDefinition, sortItems: ISortItem[]): ITotal[] {
+    const { buckets } = def;
+    const attributeBucket = bucketsFind(buckets, BucketNames.ATTRIBUTE);
+    return attributeBucket ? sanitizeBucketTotals(attributeBucket, sortItems) : [];
+}
+
+export function cloneDefTotals(def: IExecutionDefinition): ITotal[] {
+    const { sortBy } = def;
+    const totals = sanitizeDefTotals(def, sortBy);
+    return cloneDeep(totals);
+}
```

Comment:
this is somewhat misleading function name; looks like it does cloning only.. but actually also sanitizes.

also, missing doc

Reply:
good point, just realized that the function is quite pointless, it's pretty much just cloneDeep + sanitizeDefTotals, so I simply removed it